pkg/controller/vsphere/machine_scope.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 
 	machinev1 "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
-	apivshpere "github.com/openshift/machine-api-operator/pkg/apis/vsphereprovider/v1alpha1"
```

Comment:
thanks, having the typo fix in a different comment would be more appropriate and would make review easier